src/components/Dashboard/TodoList.vue

```diff
@@ -421,8 +376,8 @@
             <PaymentIncomplete :filing=item />
           </template>
 
-          <!-- is this a conversion in any status? (staff only) -->
-          <template v-else-if="isTypeConversion(item) && isRoleStaff">
+          <!-- is this a conversion in any status? -->
+          <template v-else-if="isTypeConversion(item)">
```

Comment:
Show details for all users.

Reply:
Screenshot:

![image](https://user-images.githubusercontent.com/10002889/207436941-14b4ccac-e772-4a54-9313-de75ac43598a.png)